src/Nexus.ts

```diff
@@ -649,10 +693,35 @@ class Nexus {
           return this.request(url, args, method);
         }
       }
+      if (err instanceof JwtExpiredError && this.jwtRefreshTries < param.MAX_JWT_REFRESH_TRIES) {
+        this.jwtRefreshTries++;
```

Comment:
Shouldn't jwtRefreshTries be reset once the credentials were successfully updated?

Reply:
Good spot.